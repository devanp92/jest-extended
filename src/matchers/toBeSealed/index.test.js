import matcher from './';
expect.extend(matcher);
describe('.toBeSealed', () => {
  it('passes when given sealed object', () => {
    expect(Object.seal({})).toBeSealed();
  });

  it('fails when given a non-sealed object', () => {
    expect(() => expect({}).toBeSealed()).toThrowErrorMatchingSnapshot();
  });
});
