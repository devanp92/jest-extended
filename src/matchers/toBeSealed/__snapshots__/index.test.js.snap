// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`.toBeSealed fails when given a non-sealed object 1`] = `
"<dim>expect(</><red>received</><dim>).toBeSealed(</><green>expected</><dim>)</>

Expected object to not sealed"
`;
